migrations/versions/613bf3a40948_remove_survey_name.py

```diff
@@ -0,0 +1,23 @@
+"""remove survey name
+
+Revision ID: 613bf3a40948
+Revises: ac604d2cf763
+Create Date: 2018-08-20 10:16:35.995473
+
+"""
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = '613bf3a40948'
+down_revision = 'ac604d2cf763'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    pass
+
+
+def downgrade():
+    op.drop_column('survey_name', schema='partysvc.enrolment')
```

Comment:
Is this right? I would have thought removing the column would be in the upgrade section?

Reply:
Good spot.
Just created a PR fix for it: https://github.com/ONSdigital/ras-party/pull/168
It should fix it.